lib/reporting/reports/customers/addresses.rb

```diff
@@ -18,6 +18,7 @@ def query_result
           end.values
         end
 
+        # rubocop:disable Metrics/AbcSize
         def columns
           {
             first_name: proc { |orders| orders.first.billing_address.firstname },
```

Comment:
The question is: Do we accept that column definitions are complex in nature and we don't want to change it? Or is this something we would like to solve one day?

For the former, I would add an exception to our Rubocop styleguide config to exclude the `columns` method. And for the latter I would add this to the Rubocop todo file instead of declaring it here.

I wish we could simplify the code but it would involve a major refactor of the report. So I actually think that your declaration here is the best solution! It is something we want to solve one day but we are okay with the current code. And when we refactor one day, the statement will disappear with a refactor. :+1: